lib/parse.js

```diff
@@ -45,6 +46,14 @@ var parseValues = function parseQueryStringValues(str, options) {
     var skipIndex = -1; // Keep track of where the utf8 sentinel was found
     var i;
 
+    var updateKey = function updateObjKey(key, value) {
+        if (has.call(obj, key)) {
+            obj[key] = [].concat(obj[key]).concat(value);
```

Comment:
i believe we now have `utils.combine` to use here

Reply:
thanks, I'll check

Reply:
I check and I believe that it would be resolved during merging. 
I'll merge your main branch back to this PR as soon as there is no unresolved conversation left.